ui/lib/feed/src/components/beam-feed.tsx

```diff
@@ -77,7 +78,7 @@ const BeamFeed: React.FC<BeamFeedProps> = props => {
     <>
       {!beamsReq.initialScrollState.isFetched && (
         <Stack fullWidth={true} customStyle="p-8">
```

Comment:
the p-8 here is the reason for layout shifting on widgets, you can safely remove it because you added a full width element (AntennaLoader)

Reply:
And the main reason why it happens is because you have some fixed widths on the skeletons (on the lines) so the browser tries to keep the specified width when applying the padding. It is better to use percentages when defining the widths of those lines